service/src/main/java/uk/gov/mca/beacons/api/accountholder/application/AccountHolderService.java

```diff
@@ -58,13 +67,66 @@ public Optional<AccountHolder> getAccountHolderByAuthId(String authId) {
   public Optional<AccountHolder> updateAccountHolder(
     AccountHolderId id,
     AccountHolder accountHolderUpdate
+  ) {
+    try {
+      log.info(
+        "HELLO Account holder authId is " + accountHolderUpdate.getAuthId()
+      );
+
+      Optional<User> accountHolderInAzure = getAccountHolderFromAzureAd(
+        accountHolderUpdate.getAuthId()
```

Comment:
This is dangerous. We should be taking the accountHolderId, getting the accountHolder from that from the db. Get the authId from the db account holder, then use that value to get the user from azure